Doc/reference/expressions.rst

```diff
@@ -1930,8 +1947,11 @@ Operator precedence
 The following table summarizes the operator precedence in Python, from highest
 precedence (most binding) to lowest precedence (least binding).  Operators in
 the same box have the same precedence.  Unless the syntax is explicitly given,
-operators are binary.  Operators in the same box group left to right (except for
-exponentiation and conditional expressions, which group from right to left).
+operators are binary. Operators in the same box group left to right (except for
+comparisons, including tests, exponentiation and conditional expressions, which
+group from right to left which all have the same precedence and chain from left
+to right — see section Comparisons — and exponentiation, which groups from right
+to left)
```

Comment:
Exponentiation listed twice. The exceptions don’t all have the same precedence. Partially redundant with the paragraph below.